radio@driglu4it/files/radio@driglu4it/applet.js

```diff
@@ -26,7 +31,12 @@ MyApplet.prototype = {
   _init: function(orientation, panel_height, instance_id) {
     Util.spawnCommandLine("mocp");
     Applet.TextIconApplet.prototype._init.call(this, orientation, panel_height, instance_id);
-    this.set_applet_icon_symbolic_path(GLib.get_home_dir() + "/.local/share/cinnamon/applets/" + UUID + "/icons/radio.svg");
+
+    Gtk.IconTheme.get_default().append_search_path(ICONS_DIR);
```

Comment:
Hello, @claudiux.

I know that this PR is a work in progress, but I have to point out one thing.

This call to `Gtk.IconTheme.get_default().append_search_path()` is not needed. It is already done automatically for all xlets types [since Cinnamon 2.0.x](https://github.com/linuxmint/cinnamon/commit/db4ade8188581f2c46991aaa70757235c406cb8f).

Reply:
@claudiux  @Odyseus 
  Is it not sufficient just to remove the line:
`this.set_applet_icon_name('radio');`
This allows the SVG icon to display.

Reply:
@claudiux  @Odyseus 
  Is it not sufficient just to remove the line:
`this.set_applet_icon_name('radio');`
This allows the SVG icon to display.

Reply:
Hello, @Odyseus. It is fixed, now. Thank you.

Reply:
Thank you for your reviews.